templates/workflow/editorialLinkActions.tpl

```diff
@@ -7,6 +7,17 @@
  *
  * Show editorial link actions.
  *}
+
+{if (!empty($lastDecision))}
+	<script type="text/javascript">
+		// Expected to include the script somehow like this?
+		//$(function() {ldelim}
+		//	$('.pkp_workflow_decided').pkpHandler(
+		//		'$.pkp.controllers.LinkAction_LinkActionHideButtons'
+		//	);
+		//{rdelim});
+	</script>
+{/if}
```

Comment:
As in https://github.com/pkp/pkp-lib/pull/4583/commits/9eab04a1b14de44333d409fde7b5ad5329db236a#diff-bc97fc94e30294769a544cd4a2468a95 this is just a draft where I think the js code should/could be placed Help appreciated and needed! :)